infra/gravity/provision.go

```diff
@@ -207,6 +368,9 @@ func configureVM(ctx context.Context, log logrus.FieldLogger, node infra.Node, p
 		err = bootstrapAWS(ctx, g, param)
 	case "azure":
 		err = bootstrapAzure(ctx, g, param)
+	case "ops":
+		// For ops installs, we don't run the installer. So just hardcode the install directory to /bin
+		g.installDir = "/bin"
```

Comment:
can it be left empty?

Reply:
Not easily, the gravity command appears to be called as follows:
```
cmd := fmt.Sprintf("cd %s && sudo ./gravity status --system-log-file=./telekube-system.log", g.installDir)
```

So unless I try and refactor that code to handle the case where installDir isn't set to try and run gravity from the path, installDir needs to get set somewhere.